packages/evolution-common/src/services/baseObjects/BaseTrip.ts

```diff
@@ -37,22 +37,22 @@ export type ExtendedTripAttributes = BaseTripAttributes & {[key: string]: any};
 export class BaseTrip extends Uuidable implements IValidatable {
 
     _isValid: OptionalValidity;
-    _weight?: Weight;
+    _weights?: Weight[];
```

Comment:
What do weights represent? Why an array?

Reply:
Weighting a sample over the reference population is the usual process involved when dealing with statistically based surveys. We need to make sure that the results are representative of the overall population associated with the survey. Each main object of survey data can be weighted, and each can include distinct weighting methods, so this is why we need to accept an array of weights, which includes the weight itself (a decimal number) and the weighting method. I am not sure we should include this in the documentation for now, since this is pretty usual stuff. All Origin-Destination and travel surveys are weighted.